rasa/shared/utils/schemas/stories.yml

```diff
@@ -103,6 +103,8 @@ mapping:
                 sequence:
                 - type: "map"
                   mapping: *intent_and_entities
+                - type: "map"
+                  mapping: *slot_was_set_seq
```

Comment:
What does the `*` do here? I've tried looking this up in pykwalify docs but couldn't find anything.

Reply:
This is basically how linking is done. You can check other mappings in this file (for example, `mapping: *intent_and_entities` just above this)

Reply:
for example:
```
block1: &myblock
  a: 1
  b: 2
block2:
  subblock: *myblock
```

is equivalent to:
```
block1:
  a: 1
  b: 2
block2:
  subblock:
    a: 1
    b: 2
```